src/boosting/gbdt.cpp

```diff
@@ -718,7 +719,7 @@ void GBDT::ResetConfig(const Config* config) {
   if (train_data_ != nullptr) {
     ResetBaggingConfig(new_config.get(), false);
   }
-  if (config_->forcedsplits_filename != new_config->forcedbins_filename) {
+  if (config_.get() != nullptr && config_->forcedsplits_filename != new_config->forcedbins_filename) {
```

Comment:
@guolinke I think there might be a typo here, should it be `new_config->forcedsplits_filename` (instead of `new_config->forcedbins_filename`)?

Reply:
@btrotta you are right, it is typo.

Reply:
@btrotta can you help to create a PR?

Reply:
@guolinke Done in #3392. 

@shiyu1994 Apologies, I just now realized you are the author of this PR, not guolinke. I should have mentioned you in my original comment.